### ISSUE TITLE
Upscale and stretch  videos in streaming websites

### ISSUE THREAD

**Issue by trimechee** [Eloston/ungoogled-chromium#1946]
Hello, this necessary and important extension because we watch a lot of streaming sites and often there are side black bars and this extension allows to remove them, I tried with Chromium and Ungoogled-Chromium and it didn't work, sure because this extension officially only supports firefox and google chrome, so i had to fiddle around and search and i tried one of these solutions and it worked! please can you add official support for this incredible essential extension, ie add an option to be able to change the aspect ratio of the videos in an easy way without having to manipulate the experimental flags of the browser?

 the extension : https://github.com/dvlden/ultrawideo

Upscale or stretch any video on the web, to make it look great on UltraWide screens.

The only extension on the market with two significant functionalities that no other extension in this category possess.

Works on all video streaming platforms. ✅
Works on all embedded videos. ✅
It doesn't matter if there are multiple direct or embedded video players on a single page. The extension is smart enough to manipulate the video you put into a fullscreen, without affecting other video players.

If you are using multiple browsers on the same or different devices, this extension is cross-browser and will work in all major browsers (except Safari). I tested it in Chrome, Firefox, Edge, Brave, Whale, and Opera; without a single issue. Many other browsers use Webkit engine, hence this extension will work in those browsers too.

Approach 1
Run the browser with a specific flag.

Quit the browser
Right click on the browser's shortcut and go to Properties
Within the Target box, add --disable-direct-composition-video-overlays` to the end (outside of quotation mark) and apply the changes
Launch the browser
Approach 2
Change graphics back-end for ANGLE.

Launch the browser
Open browser://flags/ (replace word browser, with the one you're using, e.g. chrome://flags/)
Search for ANGLE (full option name: Choose ANGLE graphics backend)
Choose D3D9 or OpenGL
Restart the browser
Approach 3
Disable hardware acceleration.

Launch the browser
Go to browser settings
Search for hardware (full option name: Use hardware acceleration when available)

**Comment by PF4Public** [comment]
Those approaches are not that complicated to do oneself without adding any additional; patches into ungoogled-chromium. Each additional patch increases the entropy and hardens the maintenance. One should be very careful at adding patches.

**Comment by trimechee** [comment]
yes i konw, mais ordinary user has must to investigate, search, foullier in the flags.....and UngoogleD chromium is a browser which differs from the others, for example ungoogled chromuim int-gre reader mode flag removed in chromium, here we are talking about an important feature, being able to watch videos without black band and it will be great that Ungoogled chromium integrates a native "change aspect ratio" option in order to dazzle the world even if it's very difficult, we know it :)
